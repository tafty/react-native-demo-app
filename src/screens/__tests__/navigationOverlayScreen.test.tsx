import 'react-native'
import React from 'react'
import { Provider } from 'react-redux'
import renderer from 'react-test-renderer'

import configureStore from 'redux-mock-store'
import NavigationActionOverlayScreen from '../navigationActionOverlayScreen'

const middlewares: any[] = []
const mockStore = configureStore(middlewares)

describe('NavigationActionOverlayScreen Screen', () => {
  it('Should render', () => {
    jest.useFakeTimers()

    const initialState = {}

    const store = mockStore(initialState)

    const tree = renderer.create(
      <Provider store={store}>
        <NavigationActionOverlayScreen
          componentId="1243534"
          navigationActionId="abcd"
        />
      </Provider>,
    )

    expect(tree).toMatchSnapshot()
  })
})
