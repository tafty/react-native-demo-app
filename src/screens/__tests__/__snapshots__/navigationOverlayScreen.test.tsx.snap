// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavigationActionOverlayScreen Screen Should render 1`] = `
<RCTScrollView
  style={
    Object {
      "backgroundColor": "rgba(0, 0, 0, 0)",
      "flex": 1,
    }
  }
>
  <View>
    <View
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "rgba(0, 0, 0, 0)",
          "flex": 1,
          "height": 1334,
          "justifyContent": "center",
        }
      }
    >
      <View />
    </View>
  </View>
</RCTScrollView>
`;
